SignalBackgroundMerger.cpp

```diff
@@ -237,13 +237,13 @@ class SignalBackgroundMerger {
       .help("Number of fourth background events to skip. Default is 0");
     
     args.add_argument("-o", "--outputFile")
-      .default_value(std::string("bgmerged.hepmc3.tree.root"))
-      .help("Specify the output file name. By default bgmerged.hepmc3.tree.root is used");
+      .default_value(std::string("bgmerged"))
+      .help("Specify the output file name. By default bgmerged is used");
 
-    args.add_argument("-r", "--rootFormat")
-      .default_value(true)
+    args.add_argument("-hepmc", "--hepmcFormat")
+      .default_value(false)
```

Comment:
Please don't change existing argument name and defaults. You will break scripts from I don't know how many users.

Reply:
I brought the original flag back. 

However, note that this `rootFormat` flag, with both default and implicit value set to true, essentially force the output format to `.root` because the argparse treats any argument with implicit value as a **flag** so `--rootFormat false` is not allowed. That's why I added the hepmc flag to overwrite the rootFormat value. 

Now the code should not break any existing script, but still allow the hepmc output format.